mockgen/parse_test.go

```diff
@@ -167,7 +167,10 @@ func TestParsePackageImport(t *testing.T) {
 	} {
 		t.Run(testCase.name, func(t *testing.T) {
 			for key, value := range testCase.envs {
-				os.Setenv(key, value)
+				err = os.Setenv(key, value)
+				if err != nil {
```

Comment:
nit: can we do the following for all of these:

```suggestion
				if err := os.Setenv(key, value); err != nil {
```